corehq/apps/userreports/pillow.py

```diff
@@ -45,6 +46,7 @@ def timed(*args, **kw):
                 table.config._id, doc['_id'], seconds
             )
             pillow_logging.warning(message)
+            _slow_ucr_assert(seconds < 5, message)
```

Comment:
Do we get softasserts onto Sentry. Might be easy to do these kinds of analysis on Sentry than in email